lib/std/fmt.zig

```diff
@@ -311,10 +310,11 @@ pub const Specifier = union(enum) {
     named: []const u8,
 };
 
+/// Intended for parsing std.fmt format strings without having to replicate the
```

Comment:
A doc comment should be about the functionality of the commented variable, rather than about its intention.

Something like "A stream based parser for format strings", perhaps.

But "what this does" rather than "why this is here".